VmAgent.Core/Interfaces/ISessionHostManager.cs

```diff
@@ -70,5 +70,12 @@ public interface ISessionHostManager
         bool IsStartupScriptExecutionComplete();
 
         bool IsUnassignable();
+
+        /// <summary>
+        /// Signals that a crash dump was found in the given SessionHost's logs folder, and checks
+        /// if the crash dump should be deleted due to throttling.
+        /// </summary>
+        /// <returns>True if the crash dump should be deleted, false otherwise</returns>
+        bool SignalDumpFoundAndCheckIfThrottled(string sessionHostId);
```

Comment:
Can we name it "ShouldDeleteDumpsFromLogsFolder". I am guessing it will only be called when we find a dump file, so the signal part is potentially redundant.

Reply:
I'm not happy with the current name either, but my thinking was that this method will be where we put the logic that puts the necessary info into the heartbeat for a playstream event to get fired. So this method will "signal that a dump was found". I actually did consider naming it "ShouldDeleteDumps" but that doesn't capture the fact that a signal will be sent, which might be an unintended side effect depending on where it's called.